packages/react-components/react-calendar-compat/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-calendar-compat",
+  "version": "0.0.0",
```

Comment:
Is 0.0.0 a valid version number or do you need 0.0.1?

Reply:
0.0.0 is valid but once we go public it has to be 0.1.0